contracts/v2/PolygonRollupManager.sol

```diff
@@ -531,8 +524,12 @@ contract PolygonRollupManager is
         uint32 rollupTypeID = ++rollupTypeCount;
 
         if (rollupVerifierType == VerifierType.Pessimistic) {
-            // No genesis on pessimistic rollups
-            if (genesis != bytes32(0)) revert InvalidRollupType();
+            // Those params should be zero for Pessimistic rollup types
+            if (
+                genesis != bytes32(0) ||
+                verifier != address(0) ||
+                programVKey != bytes32(0)
```

Comment:
we can remove the `programVKey` from the inputs

Reply:
Decided to remove in next upgrade among other unused params